### ISSUE TITLE
If an end-effector is not specified, it should be automatically configured from remaining links.

### ISSUE THREAD

**Issue by hersh** [moveit/moveit_ros#338]
I did not initially realize that an end-effector needed to be specified, so was frustrated trying to use the tools.  @sachinchitta says that it should actually be able to automatically decide that the remainder of the links in the urdf after the "last link" in a planning group is an end-effector.

**Comment by isucan** [comment]
I do not think this should be the case. Planning and dragging an arm around works fine without an end-effector. Pick&place does not. But rather than silently assuming sets of links are end effectors we should give an error. That happens in the command line, but perhaps we can add such a warning in more places.

**Comment by acornacorn** [comment]
I don't think it makes sense to automatically assume extra links are an end effector.  But moveit should allow IK and planning to work for chain groups whether the chain terminates in an end effector or not.
